python/py-mpd2/Portfile

```diff
@@ -35,6 +35,4 @@ if {${name} ne ${subport}} {
     test.args       -m unittest discover
```

Comment:
this should use `python.test_framework unittest` instead; see the `python` PG

Reply:
Good point, thanks